packages/components/psammead-section-label/src/index.stories.jsx

```diff
@@ -16,6 +16,10 @@ storiesOf('Components|SectionLabel', module)
           script={script}
           dir={dir}
           bar={boolean('show bar?', true)}
+          hideSectionHeader={boolean(
+            'hide first section header when width less than 600px?',
```

Comment:
The component in isolation doesnt know the `first section header` logic, maybe just `hide component when width less than 600px?`

Reply:
'Visually hide first section header when width less than 600px'

Reply:
Changed it to `visually hide component when width less than 600px?`